### DISCUSSION TITLE
docs/wintergatan-productivity-theory/

### DISCUSSION THREAD

**Discussion by giscus** [Yarden-zamir/me#10] [Announcements]
# docs/wintergatan-productivity-theory/

https://www.yarden-zamir.com/docs/wintergatan-productivity-theory/

**Comment by Yarden-zamir** [comment]
Potatoes!

**Comment by TomCohenDev** [comment]
Cucumber

**Reply by Yarden-zamir** [reply]
Agreed

**Reply by amos-joshua** [reply]
plants are just a metaphor for software
